bin/sstableloader

```diff
@@ -32,18 +35,18 @@ elif [ -r "$CASSANDRA_INCLUDE" ]; then
     . "$CASSANDRA_INCLUDE"
 fi
 
-if [ -z "$CLASSPATH" ]; then
-    echo "You must set the CLASSPATH var" >&2
-    exit 1
-fi
+SSTABLELOADER_PATH="$CASSANDRA_HOME/tools/bin/sstableloader"
 
-if [ "x$MAX_HEAP_SIZE" = "x" ]; then
-    MAX_HEAP_SIZE="256M"
+if [ ! -f "$SSTABLELOADER_PATH" ]; then
+    echo "Note: sstableloader has moved to the tools directory. \
+Detected that $SSTABLELOADER_PATH does not exist, this is likely a packaging \
```

Comment:
I don't get `this is likely a packaging issue as this script should not be installed.`

Imho should be `Note: sstableloader has moved to the tools directory. Please install cassandra-tools package and try again.`

This script should be still installed. It is just it will point to a new executable. It is just a delegate.

Reply:
hmmm thinking about it more, I see what we are doing here. This script is still part of tarball, it is just not "installed" as part of deb / rpm. When I look closely, I see this was installed to '/usr/bin/sstableloader` and `tools/bin/sstableloader` can not be installed there as well because it would replace what was already in /usr/bin/sstableloader. Huh.

But still. I don't think this is _packaging issue_. Because e.g. a developer might still execute this script as he develops.

Reply:
changed the error message and removed the else path :+1: